website/pages/en/tokenomics.mdx

```diff
@@ -75,31 +75,23 @@ Indexers are the backbone of The Graph. They operate independent hardware and so
 
-Indexers can increase their GRT allocations on subgraphs by accepting GRT delegation from Delegators, and they can accept up to 16 times their initial stake. If an Indexer becomes "over-delegated" (i.e., more than 16 times their initial stake), they will not be able to use the additional GRT from Delegators until they increase their stake in the network.
+Indexers can increase their GRT allocations on subgraphs by accepting GRT delegation from Delegators, and they can accept up to 16 times their initial self stake. If an Indexer becomes "over-delegated" (i.e., more than 16 times their initial self stake), they will not be able to use the additional GRT from Delegators until they increase their self stake in the network.
 
-The amount of rewards an Indexer receives can vary based on the initial stake, accepted delegation, quality of service, and many more factors. The following chart is publicly available data from an active Indexer on The Graph's decentralized network.
-
-### The Indexer stake & reward of allnodes-com.eth
-
-![Indexing stake and rewards](/img/indexing-stake-and-income.png)
-
-This data is from February 2021 to September 2022.
-
-> Please note, this will improve when the [Arbitrum migration](https://forum.thegraph.com/t/gip-0037-the-graph-arbitrum-deployment-with-linear-rewards-minted-in-l2/3551) is complete, making gas costs a significantly lower burden for participating on the network.
+The amount of rewards an Indexer receives can vary based on the Indexer's self stake, accepted delegation, quality of service, and many more factors.
 
 ## Token Supply: Burning & Issuance
 
 The initial token supply is 10 billion GRT, with a target of 3% new issuance annually to reward Indexers for allocating stake on subgraphs. This means that the total supply of GRT tokens will increase by 3% each year as new tokens are issued to Indexers for their contribution to the network.
 
-The Graph is designed with multiple burning mechanisms to offset new token issuance. Approximately 1% of the GRT supply is burned annually through various activities on the network, and this number has been increasing as network activity continues to grow. These burning activities include a 0.5% delegation tax whenever a Delegator delegates GRT to an Indexer, a 1% curation tax when Curators signal on a subgraph, and a 1% of query fees for blockchain data.
+The Graph is designed with multiple burning mechanisms to offset new token issuance. Approximately 1% of the GRT supply is burned annually through various activities on the network, and this number has increased as network activity continues to grow. These burning activities include a 0.5% delegation tax whenever a Delegator delegates GRT to an Indexer, a 1% curation tax when Curators signal on a subgraph, and a 1% of query fees for blockchain data.
```

Comment:
I also feel like it was more correct before. It sounds like we want to say that this number stopped increasing, but if that's the case, what's the final number? Still approximately 1%?

Reply:
I was trying to improve clarity but agreed - changing that back. Thanks for flagging!